src/reducers/entities/search.js

```diff
@@ -9,7 +9,7 @@ function results(state = [], action) {
     switch (action.type) {
     case SearchTypes.RECEIVED_SEARCH_POSTS: {
         if (action.isGettingMore) {
-            return state.concat(action.data.order);
+            return [...new Set(state.concat(action.data.order))];
```

Comment:
It's a bit strange to use a set here. The semantics of sets in general don't preserve order. I am assuming JS preserves order here.

Reply:
Yes it preserves the order

Reply:
It's definitely a bit weird to look at, but in JS, Sets are iterated in insertion order, so this is correct.